minecraft_servers/survivalgames/manifest.json

```diff
@@ -0,0 +1,31 @@
+{
+  "server_name": "survivalgames",
+  "nice_name": "SurvivalGames",
+  "direct_ip": "mc.survival-games.cz",
+  "supported_languages": [
+     "cs",
+     "sk"
+	 "en"
+  ],
+  "social": {
+    "web": "https://survival-games.cz",
+    "web_shop": "https://store.survival-games.cz",
+    "web_support": "https://account.survival-games.cz",
```

Comment:
Cannot be a login protected page; use a public faq or similar

Reply:
Okay i will ad the FAQ page.